samples/attested_tls/client/enc/client.cpp

```diff
@@ -219,6 +219,7 @@ int launch_tls_client(char* server_name, char* server_port)
     mbedtls_ctr_drbg_init(&ctr_drbg);
     mbedtls_x509_crt_init(&client_cert);
     mbedtls_pk_init(&pkey);
+    oe_verifier_initialize();
```

Comment:
Just want to call out that now the `oe_verifier_initialize` can still be implicitly called (e.g., when using `oe_verify_report`). Also, the `oe_verifier_shutdown` is always called at the enclave termination. These hold true for the `oe_attester_initialize`/`oe_attester_shutdown` as well. I'm wondering if we need to remove those implicit behaviors.

cc @dthaler @yentsanglee @CodeMonkeyLeet

Reply:
My opinion is that it is more dev friendly if the app does not need to explicitly initialize/shutdown the attester and so I would rather delete this line of code.

Reply:
Even though the initialize and the shutdown are implicitly called, I think it's a good practice to have them in the samples. Until we have an overall plugin design, external plugins have to provide similar initialization.